src/ensembl/src/shared/components/layout/DocLayout.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/common';
+
+$sidebarContentWidth: 320px;
+
+.DocLayout {
+  height: 100%;
+  min-height: 550px; // based on the current interface of genome browser
+  overflow: hidden;
+}
+
+.mainWrapper {
+  height: 100%;
+  position: relative;
+}
+
+.main {
+  height: 100%;
+  margin-left: $sidebarContentWidth;
+  margin-right: 48px;
+}
+
+.sidebarWrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  z-index: 1;
+}
+
+.sidebar {
+  flex: 0 0 auto;
+  height: 100%;
+  width: $sidebarContentWidth;
+  background-color: white;
+  padding: 0px 36px;
+
+  .runningHeader {
+    font-size: 12px;
+    font-weight: bold;
+    color: $dark-grey;
+    margin-bottom: 24px;
+  }
+
+  ul {
+    list-style: none;
+    margin-left: -40px;
```

Comment:
We probably want to remove all native browser styles (listt-style, padding-left) from the `ul` element in a global CSS file (styles/normalize.scss?). I believe we haven't used the `ul` anywhere yet; that's why it has the browser defaults on it.

Reply:
OK. That's probably a wider question, though - I don't want to mess up anyone else's layout!